gradle.properties

```diff
@@ -0,0 +1,6 @@
+//signing.keyId=<publicKey>
```

Comment:
[Here](https://www.timroes.de/2013/09/22/handling-signing-configs-with-gradle/) I found a post related to keeping credentials in properties file. Alternatively it's perhaps possible in a similar way to load a groovy script where settings are being configured.